.npmignore

```diff
@@ -0,0 +1 @@
+src/
```

Comment:
if we do `.npmignore` it should contain everything from `.gitignore` too. npm respects `.gitignore` if there is no `.npmignore`.

I'm thinking this is a potential footgun... We often store credentials for testing in a file that we added to `.gitignore`. So if we start adding `.npmignore` we really can end up accidentally submitting our secrets to npm , he he... It's actually pure luck that we haven't already done this :)

Also when debugging I often open files in `node_modules/` so maybe it's not bad to just include the source files for good measure... It's not like it'll cost anything... We have millions of files in `node_modules` already...